app/controllers/collections_list_controller.rb

```diff
@@ -0,0 +1,10 @@
+# Front-end controller to list public collections.
+class CollectionsListController < ApplicationController
+  def index
+    @collections = Collection.
+      where("published = true").
+      order(:title).
+      with_representative_derivatives
+    render :template => "collections/index"
```

Comment:
This is an unusual way to do this.  More typical ways to do this:

* `render "collections/index"` -- you don't need the `template:`, either that's the default, or maybe it even does a slightly different thing that isn't right, not sure. Normally it's done like this. EXCEPT. 

* `render "index"` or `render :index` -- it'll look up in `collections/` automatically, because that's the controller we're in. If you were using controller superclasses, if not found in `collections/` it will *also* look in a superclass, up the chain. You break this if you say `render "collections/index"` specifically.  We're not doing this at present, but avoid surprises by doing things conventionally. 

* In fact, `index` is the default since we are in an action method `def index`, so you can just say `render`, or even leave off the `render` entirely. 

The last is most conventional -- rendering a standard template with the same name as the action method, in the same folder as the controller -- leave off the `render` entirely. 

I will make the commit myself to make this change.

Reply:
Although actually you didn't have the view in the standard place. Since the controller is CollectionsListController, the view should be in `collections_list/index.html.erb`, not `collections/`. I will fix that too.

Reply:
I moved the controller and got rid of the 'render'. Thanks for the advice there!